apps/system/utils/utils_proc.h

```diff
@@ -37,6 +37,18 @@ enum proc_stat_data_e {
 	PROC_STAT_PEAKSTACK,
 	PROC_STAT_CURRHEAP,
 	PROC_STAT_PEAKHEAP,
+#ifdef CONFIG_SCHED_CPULOAD
+#ifdef CONFIG_SCHED_MULTI_CPULOAD
+	PROC_STAT_CPULOAD_SHORT,
```

Comment:
Instead of definition of period, why don't you provide more directory? (below is an example)
```
proc/cpuload/10s
            /100s
            /100s
```
Of course each of directory would be added when specific config is set, if I were you.